docs/customizing.md

```diff
@@ -245,3 +245,277 @@ Example `pulp_install_plugins` (with bogus version values):
+          NAME: pulp
+          USER: pulp
+          PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+    pulp_install_plugins:
+      pulp-container: {}
+      pulp-rpm: {}
+  roles:
+    - pulp_services
+  environment:
+    DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+```
+
+#### Pulp with Separate Servers for Services
+
+This scenario has the following layout:
+
+* Pulp on one server
+* The database on a second server
+* redis on a third server
+* The webserver on a fourth server
+
+Note that `pulp_webserver` still depends on `pulp_common` to provide the python packages for the
+per-plugin webserver snippets. This means that the webserver will have pulpcore and all plugins
+installed on it but no code will be running.
+
+```
+---
+- hosts: example-postgres-server
+  vars:
+    pulp_settings:
+      databases:
+        default:
+          HOST: example-postgres-server
+          NAME: pulp
+          USER: pulp
+          PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+  roles:
+    - pulp_database
+
+- hosts: example-redis-server
+  vars:
+    pulp_redis_bind: 'example-redis-server:6379'
+  roles:
+    - pulp_redis
+
+- hosts: example-pulp-server
+  vars:
+    pulp_default_admin_password: << YOUR PASSWORD FOR THE PULP APPLICATION HERE >>
+    pulp_content_bind: example-pulp-server:24816
+    pulp_api_bind: example-pulp-server:24817
+    pulp_settings:
+      secret_key: << YOUR SECRET HERE >>
+      content_origin: "http://example-webserver.fqdn"
+      redis_host: example-redis-server
+      redis_port: 6379
+      databases:
+        default:
+          HOST: example-postgres-server
+          ENGINE: django.db.backends.postgresql_psycopg2
+          NAME: pulp
+          USER: pulp
+          PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+    pulp_install_plugins:
+      pulp-container: {}
+      pulp-rpm: {}
+  roles:
+    - pulp_services
+  environment:
+    DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+
+- hosts: example-webserver
+  vars:
+    pulp_content_bind: example-pulp-server:24816
+    pulp_api_bind: example-pulp-server:24817
+  roles:
+    - pulp_webserver
+```
+
+#### Separate Servers for Each and Every service
+
+In this deployment, every single service is deployed on its own server.
+
+Note that `pulp_webserver` still depends on `pulp_common` to provide the python packages for the
+per-plugin webserver snippets. This means that the webserver will have pulpcore and all plugins
+installed on it but no code will be running.
+
+In this example, there are two Pulp worker servers.
+
+```
+---
+- hosts: example-postgres-server
+  vars:
+    pulp_settings:
+      databases:
+        default:
+          HOST: example-postgres-server
+          NAME: pulp
+          USER: pulp
+          PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+  roles:
+    - pulp_database
+
+- hosts: example-redis-server
+  vars:
+    pulp_redis_bind: 'example-redis-server:6379'
+  roles:
+    - pulp_redis
+
+- hosts: pulp-api1
```

Comment:
This and every other `pulp-api1`  should be renamed to `example-pulp-api-server` for the sake of consistency.